### PR TITLE
Lab on Neural Network 

### PR DESCRIPTION
**Pull request by rospiero**
Lab on Neural Network

### CODE REVIEW

**Review by rospiero — COMMENTED** [review]
image

**Comment by rospiero** [comment]
Epoch 307
Learning rate 0,03
Activation function ReLU
Features included X1, X2
Hidden layers and neurons 3 (8,8,8)
Test and training loss: 0,032 0,026
